.github/workflows/ci-tests.yml

```diff
@@ -135,7 +135,7 @@ jobs:
             -Dsonar.projectKey=TykTechnologies_tyk
             -Dsonar.sources=.
             -Dsonar.exclusions=**/testdata/*,test/**,coprocess/**/*,ci/**,smoke-tests/**,apidef/oas/schema/schema.gen.go
-            -Dsonar.coverage.exclusions=**/*_test.go,**/mock/*
+            -Dsonar.coverage.exclusions=**/*_test.go,**/mock/*,internal/crypto/testutil.go
```

Comment:
Revert this, include it in coverage

Reply:
It's a test helper and since it's a separate package it wouldn't be reported in the coverage

Reply:
let it be reported, the package is /internal and it's an importable API, let's consider coverage confirms usage